packages/core/content-type-builder/admin/src/translations/en.json

```diff
@@ -57,6 +57,10 @@
   "error.attributeName.reserved-name": "This name cannot be used in your content type as it might break other functionalities",
   "error.contentType.pluralName-used": "This value cannot be the same as the singular one",
   "error.contentType.singularName-used": "This value cannot be the same as the plural one",
+  "error.contentType.singularIDEqualsPluralID": "This value cannot be the same as the Plural API ID of another content type.",
+  "error.contentType.pluralIDEqualsSingularID": "This value cannot be the same as the Singular API ID of another content type.",
+  "error.contentTypeName.pluralIDEqualsCollectionName": "This value is already in use by another content type.",
+  "error.contentTypeName.singularIDEqualsCollectionName": "This value is already in use by another content type.",
```

Comment:
Just looking at the context of the other lines here, it looks like we should be using kebab-case instead of camelCase for the string ids. But I'm not really sure, it doesn't seem totally consistent

Reply:
I followed the same syntax of the existing labels like for example "error.contentType.pluralName-used", it is a mix of camel case and kebab-case, maybe I can try to find a good mix also for these new contents